packages/app/src/app.ts

```diff
@@ -181,8 +180,7 @@ export class App<
     if (options._locals) locals = { ...locals, ...options._locals }
 
     if (!file.endsWith(`.${options.ext}`)) file = `${file}.${options.ext}`
-
-    const dest = options.viewsFolder ? path.join(options.viewsFolder, file) : file
+    const dest = path.join(options.viewsFolder, file)
```

Comment:
why remove a ternary? it might be that views folder is set to `''`

Reply:
that is not possible due to 
```ts
options.viewsFolder = options.viewsFolder || this.settings.views || `${process.cwd()}/views`
```
always having a fall back to at least `/views` (assuming `process.cwd()` is '')